exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/partitionsender/PartitionerDecorator.java

```diff
@@ -26,13 +26,13 @@
 import org.apache.drill.exec.ops.FragmentContext;
 import org.apache.drill.exec.ops.OperatorStats;
 import org.apache.drill.exec.record.RecordBatch;
-
```

Comment:
don't have a very strong opinion on this, but there seem to be a lot of shifting of imports without actually changing them, are you standardizing the order? I don't think we have a lot of outstanding patches that will have merge issues as a result, but we generally try to avoid cosmetic changes, especially mixed in with other work.